SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.cc

```diff
@@ -976,7 +983,7 @@ int Phase2TrackerDigitizerAlgorithm::convertSignalToAdc(uint32_t detID, float si
         temp_signal = std::floor((temp_signal - kink_point) / (pow(2, dualslope_param - 1))) + kink_point;
     }
     signal_in_adc = std::min(temp_signal, theAdcFullScale_);
-    LogTrace("Phase2TrackerDigitizerAlgorithm")
+    LogInfo("Phase2TrackerDigitizerAlgorithm")
```

Comment:
does this need to be available in production jobs? if not, please revert to LogTrace